versioning_function.sql

```diff
@@ -34,7 +34,7 @@ BEGIN
   history_table := TG_ARGV[1];
 
   -- check if sys_period exists on original table
-  SELECT atttypid, attndims INTO holder FROM pg_attribute WHERE attrelid = TG_TABLE_NAME::regclass AND attname = sys_period AND NOT attisdropped;
+  SELECT atttypid, attndims INTO holder FROM pg_attribute WHERE attrelid = TG_RELID AND attname = sys_period AND NOT attisdropped;
```

Comment:
TG_TABLE_NAME isn't enough as it doesn't specify schema.

Reply:
I also noticed that if I used quoted identifiers, such as "Users", TG_TABLE_NAME would be `users` (lower cased).  This would give the error: `column "%" of relation "%" does not exist'` 

This change (and using it elsewhere) appears to fix that for the source table, but not for the history table.  I haven't made any changes, yet, to see if I could fix the history table, also.

It's not a big deal, but it is an issue since I'm just starting out with postgres.  I have since gone to the standard of lower case and underscores (or, at least, non-quoted identifiers).